Documentation/compatibility/workflow-designer-accessibility.md

```diff
@@ -0,0 +1,123 @@
+## Accessibility improvements in WF workflow designer
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The Windows Workflow Foundation (WF) workflow designer is improving how it works with accessibility technologies. These improvements include the following changes:
+- Tab order changed to left->right & top->bottom in some controls
+  - Initialize correlation window for setting correlation data for the InitializeCorrelation activity
+  - Content definition window for Receive, Send, SendReply, and ReceiveReply activities
+- More functions available via keyboard
+  - When editing the properties of an activity, fixed an issue where property groups could not be collapsed by keyboard the first time they are focused
+  - Warning icons are now accessible by keyboard
+  - More Properties button in properties window is no accessible by keyboard
+  - Keyboard users now can access the header items in the arguments and variables panes of the workflow designer
+- Improved visibility of focused items such as when
+  - Adding rows to data grids used by the workflow designer and activity designers
+  - Tabbing through fields in the ReceiveReply and SendReply activities
+  - Setting default values for variables or arguments
+- Users of screen readers will see several improvements where the screen reader can now correctly recognize
+  - Breakpoints set in the workflow designer
+  - FlowSwitch, FlowDecision, and CorrelationScope activities
+  - Contents of the Receive activity
+  - Target type for the InvokeMethod activity
+  - Exception combobox and Finally section in the TryCatch activity
```

Comment:
Exception combobox and Finally section --> The Exception combo box and the Finally section
TryCatch as a link?